### PR TITLE
Update website for v2

### PR DESCRIPTION
**Pull request by zzzFelix**
- Update documentation
- Use v2 /scanners/ directory instead of individual scanner repositories
- Remove version tag
- Inform user that they are looking at preliminary documentation, link to v1
- Align iteratec copyright in footer
- Align background image with beta alert
- Fix broken integration links in sidebar
- Add icons for new scanners

### CODE REVIEW

File: src/templates/scannerExamples.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { graphql } from "gatsby";
+import Layout from "../components/Layout";
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
+import 'react-tabs/style/react-tabs.css';
+
+const ScannerExamples = props => {
+  const path = `${props.path.substring(13)}/`;
```
Comment (J12934) [inline src/templates/scannerExamples.js]: Why 13?

File: gatsby-node.js [hunk 2]
```diff
@@ -113,3 +133,15 @@ exports.createPages = ({ graphql, actions }) => {
     );
   });
 };
+
+exports.onCreateNode = ({ node, actions }) => {
```
Comment (J12934) [inline gatsby-node.js:118]: What triggers this "onCreateNode"?

File: src/components/MenuMobile.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ const MenuMobile = props => {
       }}
     >
       {menuLinks.map(link => (
-        <Link to={link.link}>{link.name}</Link>
+        <Link to={link.link} key={link.name}>{link.name}</Link>
```
Comment (J12934) [inline src/components/MenuMobile.js:22]: Good catch 👍

File: src/templates/scannerExamples.js (hunk 1 above)
Comment (zzzFelix) [reply src/templates/scannerExamples.js]: I was removing part of the string for display purposes. The rather unorthodox approach was not necessary though. Good catch.

File: gatsby-node.js (hunk 2 above)
Comment (zzzFelix) [reply gatsby-node.js:118]: onCreateNode will be called whenever Gatsby creates or updates a node of its internal Redux store.

**Review by SirDany — APPROVED** [review]

**Pull request merged**